### DISCUSSION TITLE
Found a bug in the wallpaper (namely, in the wallpaper, not in the application)

### DISCUSSION THREAD

**Discussion by semyon0101** [rocksdanister/lively#1240] [General]
Sorry for my English.

When choosing wallpaper "fluids" and setting "random splats" it happens, as it should be, random bursts and slowly subside, but as I understand it, even if the wallpaper is not displayed when focusing on the window, but bursts appear on this if after a long focus on the window and a sharp switch to the working screen, then it becomes all white, and then slowly darkens and then goes on as the author intended, if this is not repeated again.
![image](https://user-images.githubusercontent.com/79764144/170833852-a7fdc25a-d1f3-4936-abf5-a2380b61d6cc.png)
![image](https://user-images.githubusercontent.com/79764144/170833874-9ae8bf26-2528-4468-90d3-775a7bd0377d.png)
![image](https://user-images.githubusercontent.com/79764144/170833895-c7f7e618-1b24-4de9-b053-e942f04002ea.png)

**Comment by rocksdanister** [comment]
This is because the way Lively pauses web wallpapers changed from when Fluid wallpaper was created - with latest Lively only the rendering process is paused where as the Fluid wallpaper does not have code to handle it and javascript code continues to run in the background for splat animation even when paused.

For proper fix https://github.com/rocksdanister/lively/issues/974 needs to be resolved first then Fluid wallpaper need to be updated to handle this scenario.

For now disable Random splat in Fluid or disable wallpaper pausing in Lively performance settings.

**Comment by semyon0101** [comment]
Thank you. I hope this bug will be fixed soon
